ckan/lib/plugins.py

```diff
@@ -103,20 +107,18 @@ def register_package_plugins(map):
         for package_type in plugin.package_types():
             # Create a connection between the newly named type and the
             # package controller
-
-            map.connect('%s_search' % package_type, '/%s' % package_type,
-                        controller='package', action='search')
-
-            map.connect('%s_new' % package_type, '/%s/new' % package_type,
-                        controller='package', action='new')
-            map.connect('%s_read' % package_type, '/%s/{id}' % package_type,
-                        controller='package', action='read')
-
-            for action in ['edit', 'authz', 'history']:
-                map.connect('%s_%s' % (package_type, action),
-                            '/%s/%s/{id}' % (package_type, action),
-                            controller='package',
-                            action=action)
+            blueprint = Blueprint(
```

Comment:
i have followed same approach but when executing the tests, it looks like this blueprint is not regiestered by flask_app, any idea how we could do it ?

Reply:
We need to actually register it - i/e, with `app` object. Current code block isn't working yet. I'm planning  to move call of this function(`register_package_plugins`) to initialisation of flask app, right after registering all core bluprints. Now it called from `routing` module and that has less sense as we are rather adding new blueprints, than new routes